cdap-api/src/main/java/co/cask/cdap/api/data/schema/SchemaTypeAdapter.java

```diff
@@ -14,9 +14,9 @@
  * the License.
  */
 
-package co.cask.cdap.internal.io;
+package co.cask.cdap.api.data.schema;
```

Comment:
Can you keep it in the internal package of the `cdap-api` module? It's not intended to be a public class.